substrate/client/chain-spec/src/lib.rs

```diff
@@ -40,14 +249,10 @@
 //!
 //! pub type MyChainSpec<G> = GenericChainSpec<G, MyExtension>;
 //! ```
-//!
-//! Some parameters may require different values depending on the
-//! current blockchain height (a.k.a. forks). You can use `ChainSpecGroup`
-//! macro and provided [`Forks`](./struct.Forks.html) structure to put
-//! such parameters to your chain spec.
-//! This will allow to override a single parameter starting at specific
-//! block number.
-//!
+//! Some parameters may require different values depending on the current blockchain height (a.k.a.
+//! forks). You can use `ChainSpecGroup` macro and provided [`Forks`](./struct.Forks.html) structure
+//! to put such parameters to your chain spec. This will allow to override a single parameter
+//! starting at specific block number.
```

Comment:
```suggestion
//! forks). You can use the [`ChainSpecGroup`](macro@ChainSpecGroup) macro and the provided [`Forks`]
//! structure to add such parameters to your chain spec. This will allow overriding a single
//! parameter starting at a specific block number.
```